tutor/specs/components/course-settings/__snapshots__/student-access.spec.jsx.snap

```diff
@@ -8,7 +8,7 @@ exports[`Course Settings, student access can switch to links 1`] = `
     Choose how students access OpenStax Tutor. Access settings cannot be changed after students begin to enroll.
   </p>
   <a
-    href="http://openstax.force.com/support/articles/FAQ/What-is-the-difference-between-using-a-direct-link-and-using-LMS-integration-to-give-your-students-access-to-OpenStax-Tutor/?q=LMS+integration&l=en_US&c=Products%3ATutor&fs=Search&pn=1"
+    href="https://openstax.secure.force.com/help/articles/FAQ/What-is-the-difference-between-using-a-direct-link-and-using-LMS-integration-to-give-your-students-access-to-OpenStax-Tutor?"
```

Comment:
Is `?` at the end of a url without any params valid?

Reply:
yeah it's valid, but isn't needed. I'll remove it just so they're neater

Reply:
oh and these are snapshots for testing, they're like the VCR cassettes